.devcontainer/devcontainer.json

```
@@ -0,0 +1,33 @@
+// For format details, see https://aka.ms/devcontainer.json. For config options, see the
+// README at: https://github.com/devcontainers/templates/tree/main/src/javascript-node
+{
+    "name": "Node.js",
```

Just a minor favour - could you change this to the name of the repo instead?  I find it helps me keep track of what devcontainer I'm in:
```suggestion
    "name": "repo-file-sync-action",
```